ipwb/replay.py

```diff
@@ -672,9 +676,23 @@ def handler(signum, frame):
     return resp
 
 
+def generateNoMementosInterface_noDatetime(urir):
+    msg = '<h1>ERROR 404</h1>'
+    msg += 'No capture(s) found for {0}.'.format(urir)
+
+    msg += '''
+    <form method="get" action="/memento/*/" style="margin-top: 1.0em;">
+      <input type="text" value="{0}" id="url" name="url" aria-label="Enter a URI" />
+      <input type="submit" value="Search URL in the archive"/>
+    </form>'''.format(urir)
```

Comment:
I am wondering, should we keep this form sans-JS or make it behave the same way landing page does. Alternatively, should we server the original landing page with 404 status along with a clear notice text on it to avoid duplicating logic?

Reply:
I am not sure. I found that I would need to replicate a lot of the logic of the search box on the main page but opted for a simpler JS-less solution here.

Reply:
We might want to revisit it. I can think of the template of landing page having a well-known flash message option that receives custom messages based on certain states.

Reply:
Yes, definitely. The .py code I added has some duplicated logic and a suboptimal name. This fix is to get it working then we can improve it.

Reply:
A lot of duplication has to do with its organic growth. Perhaps separating views from data object and logic is necessary at some point.